docker scripts/unix/add-company-to-all-buckets-without-company.sh

```diff
@@ -0,0 +1,9 @@
+echo "HTEC group pib:"
+echo -e "\t105793138\n"
+echo "Pib of the company you want to add buckets to"
+read -p "pib: " pib
+echo "--DB password"
+docker exec -it skynest-db mysql -u user -p --execute="USE skynest;
```

Comment:
Why this is not written as a migration script with flyway?

Reply:
but since it is shell script, nothing stops me from executing it again right?
Every time when you need one-time script execution in database, use flyway migration because flyway it self has a mechanism that allows the scripts to be executed just once.

Reply:
Thats right. Okay, makes sense. It will be executed like flyway script just once if there is need. I'll change it to flyway script. :)